plugin/build_baron.go

```diff
@@ -119,6 +115,9 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 	}, nil
 }
 
+// IsWebhookConfigured webhook will can be retrieved from project or admin config depending on PluginAdminPageDisabled flag
```

Comment:
Actually sorry can you clean this up a bit? And with multiline comments, it's best to use full sentences/punctuation rather than relying on linebreaks